Microsoft.Toolkit.Uwp.UI.Controls/MasterDetailsView/MasterDetailsView.xaml

```diff
@@ -0,0 +1,85 @@
+
+    <Style TargetType="controls:MasterDetailsView" >
+        <Setter Property="Background" Value="{ThemeResource ApplicationPageBackgroundThemeBrush}"/>
+        <Setter Property="BorderBrush" Value="{ThemeResource ApplicationForegroundThemeBrush}"/>
+        <Setter Property="BorderThickness" Value="0"/>
+        <Setter Property="Template">
+            <Setter.Value>
+                <ControlTemplate TargetType="controls:MasterDetailsView">
+                    <Border Background="{TemplateBinding Background}"
+                            BorderBrush="{TemplateBinding BorderBrush}"
+                            BorderThickness="{TemplateBinding BorderThickness}">
+                        <VisualStateManager.VisualStateGroups>
+                            <VisualStateGroup x:Name="SelectionStates">
+                                <VisualState x:Name="NoSelectionWide"/>
+                                <VisualState x:Name="HasSelection">
+                                    <VisualState.Setters>
+                                        <Setter Target="NoSelectionPresenter.Visibility" Value="Collapsed"/>
+                                    </VisualState.Setters>
+                                </VisualState>
+                                <VisualState x:Name="NoSelectionNarrow">
+                                    <VisualState.Setters>
+                                        <Setter Target="NoSelectionPresenter.Visibility" Value="Collapsed"/>
+                                    </VisualState.Setters>
+                                </VisualState>
+                            </VisualStateGroup>
+                            <VisualStateGroup x:Name="WidthStates">
+                                <VisualState x:Name="NarrowState">
+                                    <VisualState.StateTriggers>
+                                        <AdaptiveTrigger MinWindowWidth="320" />
+                                    </VisualState.StateTriggers>
+                                    <VisualState.Setters>
+                                        <Setter Target="MasterColumn.Width" Value="*" />
+                                        <Setter Target="DetailsColumn.Width" Value="0" />
+                                        <Setter Target="DetailsPanel.(Grid.Column)" Value="0" />
+                                        <Setter Target="NoSelectionPresenter.Visibility" Value="Collapsed"/>
+                                        <Setter Target="MasterPanel.BorderThickness" Value="0"/>
+                                        <Setter Target="MasterPanel.Width" Value="NaN"/>
+                                    </VisualState.Setters>
+                                </VisualState>
+                                <VisualState x:Name="WideState">
+                                    <VisualState.StateTriggers>
+                                        <AdaptiveTrigger MinWindowWidth="720" />
```

Comment:
I've never been able to track down what the true "best practice" is on window width breakpoints, and it seems everyone does it however they want. That being said, the Windows guidelines pretty clearly denote breakpoints of 640 and below as narrow, 641 to 1007 as as medium, and 1008 and greater as large. I'm not sure where these values came from, but they do suggest apps follow them. So, following that, it seems this value should be 641.

That being said, the diagram directly above it show different breakpoints, and the header at the top shows a different set of breakpoints altogether.

Thoughts from the Microsoft folks? (@deltakosh, etc)